spec/controllers/sign_in_security_check_failed_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe SignInSecurityCheckFailedController do
+  let(:user) { build_stubbed(:user, :fully_registered) }
+
+  before do
+    stub_sign_in(user)
+  end
+
+  it 'renders the show template' do
+    stub_analytics
+
+    get :show
+
+    expect(@analytics).to have_logged_event(:sign_in_security_check_failed_visited)
+
+    expect(response).to render_template :show
+  end
```

Comment:
I think it'd be good if this test described captured the fact that we're interested in the event being logged. I also don't know that it's very valuable to assert the template being rendered, since that's the default behavior in Rails. We should also follow the pattern of having a `describe` block containing all of the test cases for a single action, so that this file can scale to accommodate more actions down the line if we choose to add any.

```suggestion
  describe '#show' do
    it 'logs an event' do
      stub_analytics

      get :show

      expect(@analytics).to have_logged_event(:sign_in_security_check_failed_visited)
    end
  end
```